packages/components/app/styles/components/pagination.scss

```diff
@@ -0,0 +1,177 @@
+//
+// PAGINATION
+//
+
+@use "../mixins/focus-ring" as *;
+
+// TODO: Move (Create component-level design tokens)
+:root {
+  // control - button-number, button-arrow:
+  --token-pagination-control-height: 36px;
+  --token-pagination-control-padding-horizontal: 12px;
+  --token-pagination-control-focus-inset: 5px;
+  // indicator for current page:
+  --token-pagination-indicator-height: 2px;
+  --token-pagination-indicator-width: 19px;
+}
+
+// * Pagination Parent (wrapper) component
+// ----------------------------------------
+.hds-pagination {
+  display: grid;
+  grid-template-areas: "left center right";
+  grid-template-rows: auto;
+  grid-template-columns: 1fr auto 1fr;
+  align-items: center;
+  margin: 0 auto;
+}
+
+// * Sub-component: Info
+// ---------------------------------------
+.hds-pagination-info {
+  grid-area: left;
+  justify-self: flex-start;
+  white-space: nowrap;
+}
+
+// * Sub-component: Nav
+// ----------------------------------------
+.hds-pagination-nav {
+  display: flex;
+  grid-area: center;
+}
+
+.hds-pagination-nav__page-list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.hds-pagination-nav__page-item { list-style-type: none; }
+
+// nav controls: button-arrow, button-number, ellipsis:
+.hds-pagination-nav__control,
+.hds-pagination-nav__ellipsis {
+  display: flex;
+  align-items: center;
+  padding: 0 var(--token-pagination-control-padding-horizontal);
+}
+
+.hds-pagination-nav__control {
+  height: var(--token-pagination-control-height);
+  text-decoration: none;
+  background: transparent;
+  border: none;
```

Comment:
if this is a button, this should be 1px solid transparent (so the height needs to be calculated: `height: calc(var((--token-pagination-control-height) - 2px)`

Reply:
Why do they need a transparent border? Should they display a border on hover or something? I don't see something like that in the Figma mocks.

I tried making the above changes and the buttons were too short.

Reply:
> Why do they need a transparent border? Should they display a border on hover or something? I don't see something like that in the Figma mocks.

It's because of accessibility (something I learned from @MelSumner 🙏): it's for users that may use high-contrast mode/settings in their browser.

> I tried making the above changes and the buttons were too short.

Yes, now looking at the actual CSS code is slightly more complex (but not too complex)
- you can add a 1px border without changing the actual `height` value (since there is no padding, and the button has `box-sizing: border-box`
- but you have to change the position of the focus ring, reducing its "inset" value by 1px (the position absolute is calculated from the inside of the border): `--token-pagination-control-focus-inset: 4px;`

Reply:
I pushed up a number of changes to add the transparent border without shifting the rest of the layout.